src/domain/location/LocationUtils.ts

```diff
@@ -21,4 +23,15 @@ const formatLocation = (data: Maybe<GraphQLLocation>): Partial<LocationModel> |
   };
 };
 
-export { formatLocation };
+export const formatDatabaseLocation = (
+  data: Partial<LocationModel> | undefined
+): { __typename?: 'Location'; city: string; country: string } | undefined => {
+  if (!data) {
+    return undefined;
+  }
+
+  return {
+    city: data?.city || '',
```

Comment:
`data` is defined at this point - no need for safe navigator here, nor in `formatLocation`.